codex/validation.nim

```diff
@@ -23,6 +24,9 @@ type
     proofTimeout: UInt256
     config: ValidationConfig
 
+const
+  MaxStorageRequestDuration: times.Duration = initDuration(days = 30)
```

Comment:
BTW: it is not just change to `30.days`. The whole type changes as it comes from different package (it is from `chronos` and not from `std/times`). So to later do:

```nim
duration.inSeconds
```

we need to use:

```nim
duration.secs

# or
duration.seconds
```

I guess because we use `chronos` everyday we have preference to use it over std lib.